website/client-old/js/services/userServices.js

```diff
@@ -190,6 +190,8 @@ angular.module('habitrpg')
         })
       }
 
+      //Calling this multiple times rapidly will not persist changes
```

Comment:
Are changes not persisted locally or on the server? Locally it shouldn't happen as this is a normal sync function